functions/tab.terminator.split.fish

```diff
@@ -0,0 +1,3 @@
+function tab.terminator.split -a cdto
+  /usr/share/terminator/remotinator vsplit
```

Comment:
can you tell me what's going on here?

what happens if it's not installed in `/usr/share` on someone's machine? is there a way to discover the actual location?

is it possible to add `$cdto` and `$cmd` support with terminator? if not, we should document this.

Reply:
@bobthecow I've remove the path to `remotinator` since it seems that the latest versions of terminator puts it in the binpath.

Yeah, unfortunately `remotinator` doesn't support taking the `$cdto` or `$cmd` args. I'll document it.

Reply:
https://github.com/gnome-terminator/terminator/issues/446 looks relevant to this conversation